README.md

```diff
@@ -3,4 +3,11 @@ Slackbot project implemented by [Kiwi.com](Kiwi.com) interns
 
 ## WIP
 - first internship crew started on this 21th April 2017
-- MVP is going to be available at the end of May 2017
\ No newline at end of file
+- MVP is going to be available at the end of May 2017
+
+## How to set up
+
+* Make sure you have `pip-tools`, `docker` and `docker-compose`.
```

Comment:
The next lines explain why it is suggested - the image building uses requirements.txt, which can be obtained using `pip-compile`. In theory, the requirements.txt file should be in a good shape, but there is a low, but non-zero probability that a person that wants to build the image may need to regenerate it (e.g. when a particular version of a dependency is deleted from pypi).
I agree that it is a dependency for building with low likelihood of being needed, so if the general consensus is to remove this mention, I am comfortable removing it.